### PR TITLE
Add script to compile JIT on x86

### PR DESCRIPTION
**Pull request by blasten**
## Description

`build_and_run_android_tests.sh` will be called from the LUCI recipe. The scenario will run on an x86 emulator, so JIT is required.

## Related Issues

https://github.com/flutter/flutter/issues/55326

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [contributor guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [C++, Objective-C, Java style guides] for the engine.
- [x] I read the [tree hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation.
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[contributor guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[tree hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: testing/scenario_app/README.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ compared against golden reside.
 
 ## Running for Android
 
+The test is run on a x86 emulator. To run the test locally, you must create an emulator running API level 28, and set the following screen settings in the avd's `config.ini` file:
```
Comment (godofredoc) [inline testing/scenario_app/README.md:45]: I think there is an ongoing effort to move to api level 29 do we want to update this to 29?

File: testing/scenario_app/compile_android_jit.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
```
Comment (godofredoc) [inline testing/scenario_app/compile_android_jit.sh:2]: 2020?

Comment (blasten) [reply testing/scenario_app/compile_android_jit.sh:2]: I think the license banner consistently uses 2013. https://github.com/flutter/engine/search?p=5&q=Copyright+2013&unscoped_q=Copyright+2013

File: testing/scenario_app/README.md (hunk 1 above)
Comment (blasten) [reply testing/scenario_app/README.md:45]: This sounds doable. I can coordinate that the Chrome infra folks.

**Pull request merged**